epidose/device/util.sh

```diff
@@ -186,6 +196,24 @@ get_new_filter()
   fi
 }
 
+# Check and update software if necessary
+# preconditions: WiFi should be turned on
+check_for_updates()
+{
+  log "Checking for updates"
+  remote_updates_available=false
+ 
+  # Set username and email because git asks for it
+  git config --system user.name "$USERNAME"
+  git config --system user.email "$EMAIL"
+  git -C "$LOCAL_REPO" remote update && git -C "$LOCAL_REPO" status -uno | grep -q 'Your branch is behind' && remote_updates_available=true
+  if [ "$remote_updates_available" = true ]; then
```

Comment:
Please break up this long line and test it directly in the `if`.

Reply:
Done.